src/Extractor/TikaServerTextExtractor.php

```diff
@@ -36,20 +43,23 @@ public function getClient()
         return $this->client ?:
             ($this->client =
                 Injector::inst()->createWithArgs(
-                    'TikaRestClient',
+                    TikaRestClient::class,
                     array($this->getServerEndpoint())
                 )
             );
     }
 
+    /**
+     * @return string
+     */
     public function getServerEndpoint()
     {
-        if (defined('SS_TIKA_ENDPOINT')) {
-            return SS_TIKA_ENDPOINT;
+        if ($endpoint = Environment::getEnv('SS_TIKA_ENDPOINT')) {
+            return $endpoint;
         }
 
-        if (getenv('SS_TIKA_ENDPOINT')) {
-            return getenv('SS_TIKA_ENDPOINT');
+        if ($endpoint = getenv('SS_TIKA_ENDPOINT')) {
+            return $endpoint;
```

Comment:
We can just remove this condition now, we won't support constants any more - just environment variables, and that's handled above.

We could support configuration instead if you think that might be a good idea?

Reply:
Need to use SilverStripe `Environment::getEnv()` too instead of php's `getenv`